### DISCUSSION TITLE
Blocked legitimate sites

### DISCUSSION THREAD

**Discussion by ghost** [DandelionSprout/adfilt#527] [Q&A] [answered]
Hello!
I been trying to figure out what was going on for days with my wireguard duckdns.org address, FINALLY traced it down to this blocklist. I used AdGuard and this is one of my list on it. Why is some duckdns addresses being blocked? I set up too different addresses using duckdns and they were both blocked. I looked through the list and my specific address was not singled out. Are all duckdns address blocked? The main https://duckdns.org is not but it looks like https://(somethinghere).duckdns.org are.

**Comment by DandelionSprout** [comment, marked as answer]
The circumstances around the blocking have been mostly lost to time by now, such that there's grounds to unblock all of `duckdns.org` again.

It was something along the lines of me believing sometime in 2021 that `DuckDNS` was an untrustworthy site-hosting that was used to mass-register malware sites, but it later turned out their legitimate use was *far* larger than what was initially believed.

The site will be fixed in `26February2022v1`, so I'll mark this as the answer.

**Reply by ghost** [reply]
Ah! Okay, that makes sense. I made an execption for my specific domains. That Fixed the issue. From what i have seen with duckdns address i have not seen any malcious sites. Then again im not a expert, but thank you for letting me know i was not crazy thinking it was this block list! XD
